priv/posts/2021/05-29-call-your-mom-app-made-in-24-hours.md

```diff
@@ -2,7 +2,8 @@
   title: "Call Your Mom app made in 24 hours for #Hack4Health",
   tags: ~w(react_native),
   description: "My experience building a React Native app in 24 hours",
-  cover_image: "05-29-call-your-mom-app-made-in-24-hours.png"
+  cover_image: "05-29-call-your-mom-app-made-in-24-hours.png",
+  author: "Brooklin Mayers"
```

Comment:
Brooklin Myers*

Also, I think we can omit the author from `.md` files unless it's a guest writer.
Then we can use a default value of `"Brooklin Myers"` in the `Post` struct.

Reply:
Oh sorry for misspelling your name  =)